person.rb

```diff
@@ -27,6 +30,10 @@ def can_use_services
     end
   end
 
```

Comment:
- [ ] please, kindly add back the method for adding rentals in the book and person class. please, the previous reviewer asked you to remove the line with `rentals << rental` and not the whole method. Nice work. Please implement back in both classes. 😄